lib/2wp-utils.js

```diff
@@ -64,7 +64,7 @@ const sendTxToBridge = async (rskTxHelper, amountInRbtc, rskFromAddress, mine =
     const txPromise = rskTxHelper.sendTransaction({
       from: rskFromAddress,
       to: BRIDGE_ADDRESS,
-      value: Number(btcEthUnitConverter.btcToWeis(amountInRbtc)),
+      value: Number(btcEthUnitConverter.ethToWeis(amountInRbtc)),
```

Comment:
Why `ethToWeis` if you pass the amount in `RBTC`? Am I missing something?

Reply:
RBTC is the same as ETH. Anyways, this was updated with a different logic.